.goreleaser.yml

```diff
@@ -0,0 +1,19 @@
+# .goreleaser.yml
```

Comment:
is that a tool ? goreleaser ?

Reply:
It is yep, this line can go though

Reply:
I think it would be good to have a setup in the make file that installed these tools:
```
setup:
    @go get <thing>
    @go get <other thing>
```